Nakama/Source/NakamaTests/Public/NakamaTestBase.h

```diff
@@ -1,8 +1,17 @@
-﻿#pragma once
+#pragma once
 
 #include "CoreMinimal.h"
 #include "NakamaClient.h"
 #include "NakamaLogger.h"
+#include "Misc/EngineVersionComparison.h"
+
+#if UE_VERSION_OLDER_THAN(5, 5, 0)
+#define NAKAMA_MODULE_TEST_MASK EAutomationTestFlags::ApplicationContextMask | EAutomationTestFlags::ProductFilter
+#else
+#include "Misc/AutomationTest.h"
```

Comment:
so only this include is conditional?  both branches look the same otherwise and can be removed from under condition?

Reply:
No, the main issue here is that they removed ApplicationContextMask from the EAutomationTestFlags enum and moved its definition to another header as a bitwise addition of some of the automation test flags. Look closely to the symbol that separates the flag enum from the flag define, "::" changed for a "_"